hub/integrations/pytorch.py

```diff
@@ -186,11 +188,18 @@ def _np_from_chunk_list(self, index: int, key: str, chunks: List[bytes]):
         shape = index_entry["shape"]
 
         combined_bytes = join_chunks(chunks, start_byte, end_byte)
+
+        def _decompress_combined_bytes(combined_bytes: Union[bytes, memoryview]):
```

Comment:
```suggestion
        def _decompress_combined_bytes(combined_bytes: memoryview):
```

can we please do this? operating on raw `bytes` is extremely slow

Reply:
also if you do this, you can remove this method def from inside this method body. we should try to never do defs within defs if we can help it